src/Microsoft.Health.Dicom.Core/Features/Query/QueryParser.TagValueParser.cs

```diff
@@ -76,5 +105,15 @@ private static DateTime ParseDate(string date, string tagName)
 
             return parsedDate;
         }
+
+        private static DateTime ParseDateTime(string dateTime, string tagName)
+        {
+            if (!DateTime.TryParseExact(dateTime, DateTimeTagValueFormat, null, System.Globalization.DateTimeStyles.None, out DateTime parsedDateTime))
```

Comment:
Based on standard, there are multiple formats we need to support, do we want to align with that?
BTW, seems FODicom already have all supported formats: https://github.com/fo-dicom/fo-dicom/blob/6be9d198d05913c00b3c2f866cd51471064417aa/FO-DICOM.Core/DicomElement.cs#L914

Standard: http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_6.2.html

Reply:
Yes, i'll add that as well with additional tests. This was mostly to show the base addition.

Reply:
Actually, I've spent more time reviewing other formats while testing and I have many concerns that I'd like to bring up during standup.